src/easystar.js

```diff
@@ -510,12 +506,16 @@ EasyStar.js = function() {
     };
 
     var getTileCost = function(x, y) {
-        return pointsToCost[x + '_' + y] || costMap[collisionGrid[y][x]]
+        return (pointsToCost[y] && pointsToCost[y][x]) || costMap[collisionGrid[y][x]]
     };
 
     var coordinateToNode = function(instance, x, y, parent, cost) {
-        if (instance.nodeHash[x + "_" + y]!==undefined) {
-            return instance.nodeHash[x + "_" + y];
+        if (instance.nodeHash[y]!==undefined) {
```

Comment:
Would you mind adding spaces around `!==`? I need to get around to adding a linter to this codebase.